dashboard-tips/shared.py

```diff
@@ -1,6 +1,6 @@
 from pathlib import Path
 
-import pandas as pd
+import polars as pl
 
 app_dir = Path(__file__).parent
-tips = pd.read_csv(app_dir / "tips.csv")
+tips = pl.scan_csv(app_dir / "tips.csv")
```

Comment:
Let's keep the data manipulation code as simple as possible for this template (i.e., avoid all the `.collect()`s)


```suggestion
tips = pl.read_csv(app_dir / "tips.csv")
```

Reply:
Is there a particular one you're thinking? Could add that to a future to-dos.

Reply:
No, I just mean that if we feel like it's that important, then we should have a template dedicated to it (i.e., create a new one that actually makes good use of lazy/large data)